setup.py

```diff
@@ -1,107 +1,40 @@
-# USAGE NOTES
-#
-# Make a PyPI release tarball with:
-#
-#     python setup.py sdist
-#
-# Upload to test PyPI with:
-#
-#     twine upload --repository-url https://test.pypi.org/legacy/ dist/*
-#
-# Install from test PyPI with:
-#
-#     pip install --index-url https://test.pypi.org/simple/ labscript_utils
-#
-# Upload to real PyPI with:
-#
-#     twine upload dist/*
-#
-# Build conda packages for all platforms (in a conda environment with setuptools_conda
-# installed) with:
-#
-#     python setup.py dist_conda
-#
-# Upoad to your own account (for testing) on anaconda cloud (in a conda environment with
-# anaconda-client installed) with:
-#
-#     anaconda upload --skip-existing conda_packages/*/*
-#
-# (Trickier on Windows, as it won't expand the wildcards)
-#
-# Upoad to the labscript-suite organisation's channel on anaconda cloud (in a
-# conda environment with anaconda-client installed) with:
-#
-#     anaconda  upload -u labscript-suite --skip-existing conda_packages/*/*
-#
-# If you need to rebuild the same version of the package for conda due to a packaging
-# issue, you must increment CONDA_BUILD_NUMBER in order to create a unique version on
-# anaconda cloud. When subsequently releasing a new version of the package,
-# CONDA_BUILD_NUMBER should be reset to zero.
-
 import os
 from setuptools import setup
 from distutils import sysconfig
 
 try:
     from setuptools_conda import dist_conda
-except ImportError:
-    dist_conda = None
 
-SETUP_REQUIRES = ['setuptools', 'setuptools_scm']
-
-INSTALL_REQUIRES = [
-    "setuptools_scm",
-    "importlib_metadata >=1.0;      python_version < '3.8'",
-    "pywin32;                       sys_platform == 'win32'",
-    "numpy >=1.15",
-    "scipy",
-    "h5py >=2.9",
-    "qtutils >=2.2.3",
-    "zprocess >=2.18.0",
-    "pyqtgraph >=0.11.0rc0;         python_version >= '3.8'",
-    "pyqtgraph;                     python_version < '3.8'",
-]
+    CMDCLASS = {"dist_conda": dist_conda}
+except ImportError:
+    CMDCLASS = {}
 
-if 'CONDA_BUILD' in os.environ:
+if "CONDA_BUILD" in os.environ:
```

Comment:
@chrisjbillington is there a reason this package doesn't override `install_requires` like the others?

Reply:
Don't remember, but probably because all of its dependencies have conda package (zprocess and qtutils on my personal conda repo, anyway). That makes me think I only ever overrode `setup_requires` for this reason, and that we should just not overwrite it and work on getting everything on conda.